validatedeps.js

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env node
```

Comment:
ok - thats fine, just don't like a proliferation of files at the root - bin dir can be used for odds and ends and scripts